rsocket-core/src/main/java/io/rsocket/frame/ErrorFrameFlyweight.java

```diff
@@ -68,4 +76,16 @@ public static ByteBuf data(ByteBuf byteBuf) {
   public static String dataUtf8(ByteBuf byteBuf) {
     return data(byteBuf).toString(StandardCharsets.UTF_8);
   }
+
+  public interface RSocketErrorCode {
```

Comment:
Lets create a class for toplevel RSocketException

Reply:
Done.